CHANGELOG.md

```diff
@@ -10,10 +10,21 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ### General
 
-- First stable release of the Splunk Distribution of OpenTelemetry Java.
+- OpenTelemetry Java SDK and OpenTelemetry Instrumentation for Java dependencies were updated to version 1.3.0.
 
 ## v1.0.0 - 2021-06-02
```

Comment:
This should be a new section for 1.1.0 release, right?

Reply:
There will be, the `./pre-release.sh` adds it automatically - I'll run it and post a separate PR